File: spunwords.js
function spinWords(string) {
  let string_array = string.split(" ");
  let new_string_array = [];
  for (let i = 0; i < string_array.length; i++) {
    if (string_array[i].length >= 5) {
      let rev_word = string_array[i].split("").reverse().join("");
      new_string_array.push(rev_word);
    } else if (string_array[i].length < 5) {
      new_string_array.push(string_array[i]);
    }
  }
  return new_string_array.join(" ");
}

let res = spinWords("Hey fellow warriors");

console.log(res);

File: spunwords.py
def spin_words(sentence):
    new_list = sentence.split(" ")
    rev_list = []
    for i in new_list:
        if len(i) >= 5:
            rev_list.append("".join(i[::-1]))

        else:
            rev_list.append(i) 
    return " ".join(rev_list)

res = spin_words("Hey fellow warriors")

print(res)